docs/installation.rst

```diff
@@ -120,6 +120,11 @@ Viewing the local server
 
 * Visit http://local.treeherder.mozilla.org in your browser. Note: There will be no data to display until the ingestion tasks are run.
 
+Serving the UI build from the dist directory (optional)
```

Comment:
I don't think we want to make the installation page more complex. I think it should be the bare minimum, with links to other pages for "common tasks" etc.